src/openforms/contrib/customer_interactions/tests/test_update_customer_interaction_data.py

```diff
@@ -0,0 +1,115 @@
+from django.test import TestCase
+
+from openforms.formio.typing.custom import DigitalAddress, SupportedChannels
+from openforms.forms.tests.factories import FormVariableFactory
+from openforms.prefill.contrib.customer_interactions.plugin import PLUGIN_IDENTIFIER
+from openforms.submissions.tests.factories import SubmissionFactory
+from openforms.utils.tests.vcr import OFVCRMixin
+from openforms.variables.constants import FormVariableDataTypes
+
+from ..utils import update_customer_interaction_data
+from .mixins import CustomerInteractionsMixin, ExpectedDigitalAddress
+
+
+class UpdateCustomerInteractionDataTests(
+    OFVCRMixin, CustomerInteractionsMixin, TestCase
+):
+    def test_non_auth_user(self):
+        profile_channels: list[SupportedChannels] = ["email", "phoneNumber"]
+        profile_data: list[DigitalAddress] = [
+            {"address": "some@email.com", "type": "email"},
+            {"address": "0612345678", "type": "phoneNumber"},
+        ]
+        submission = SubmissionFactory.from_components(
+            [
+                {
+                    "key": "profile",
+                    "type": "customerProfile",
+                    "label": "Profile",
+                    "digitalAddressTypes": profile_channels,
+                    "shouldUpdateCustomerData": True,
+                }
+            ],
+            submitted_data={
+                "profile": profile_data,
+            },
+            public_registration_reference="OF-12345",
+            form__name="With profile",
+        )
+        FormVariableFactory.create(
+            key="communication-preferences",
+            form=submission.form,
+            user_defined=True,
+            data_type=FormVariableDataTypes.array,
+            prefill_plugin=PLUGIN_IDENTIFIER,
+            prefill_options={
+                "customer_interactions_api_group": self.config.identifier,
+                "profile_form_variable": "profile",
+            },
+        )
+
+        result = update_customer_interaction_data(submission, "profile")
+
+        klantcontact = result["klantcontact"]
+        betrokkene = result["betrokkene"]
+        onderwerpobject = result["onderwerpobject"]
+        digital_addresses = result["digital_addresses"]
+
+        self.assertEqual(klantcontact["kanaal"], "open forms")
+        self.assertEqual(klantcontact["onderwerp"], "Form With profile")
```

Comment:
```suggestion
        self.assertEqual(klantcontact["onderwerp"], "With profile")
```

Surely we can use just the form name instead of adding a prefix?